Vectors.Rmd

```diff
@@ -236,7 +236,7 @@ as.integer(c("1", "1.5", "a"))
 ## Attributes {#attributes}
 \index{attributes}
 
-You might have noticed that the set of atomic vectors does not include a number of important data structures like matrices and arrays, factors and date/times. These types are built on top of atomic vectors by adding attributes. In this section, you'll learn the basics of attributes, and how the dim attribute makes matrices and arrays. In the next section you'll learn how the class attribute is used to create S3 vectors, including factors, dates, and date-times.
+You might have noticed that the set of atomic vectors does not include a number of important data structures like matrices and arrays, factors and dates and times. These types are built on top of atomic vectors by adding attributes. In this section, you'll learn the basics of attributes, and how the dim attribute makes matrices and arrays. In the next section you'll learn how the class attribute is used to create S3 vectors, including factors, dates, and date-times.
```

Comment:
```suggestion
You might have noticed that the set of atomic vectors does not include a number of important data structures like matrices and arrays, factors, and dates and times. These types are built on top of atomic vectors by adding attributes. In this section, you'll learn the basics of attributes, and how the dim attribute makes matrices and arrays. In the next section you'll learn how the class attribute is used to create S3 vectors, including factors, dates, and date-times.
```

Reply:
On second glance I realise I was using "date/times" to mean dates, or date-times